libfwupdplugin/fu-udev-device.c

```diff
@@ -1747,6 +1747,51 @@ fu_udev_device_get_devtype (FuUdevDevice *self)
 #endif
 }
 
+/**
+ * fu_udev_device_get_siblings_with_subsystem
+ * @self : a #FuUdevDevice
+ * @subsystem: the name of a udev subsystem
+ *
+ * Get a list of devices that are siblings of self and have the
+ * provided subsystem.
+ *
+ * Returns (element-type FuUdevDevice) (transfer full): devices
+ *
+ * Since: 1.6.0
+ */
+GPtrArray *
+fu_udev_device_get_siblings_with_subsystem (FuUdevDevice *self,
+					    const gchar *const subsystem)
+{
+	g_autoptr(GPtrArray)
+		out = g_ptr_array_new_with_free_func (g_object_unref);
```

Comment:
i think just a long line here, I'm fine with the column limit being broken when required